web/src/components/FilingsCalendar.test.tsx

```diff
@@ -51,13 +52,19 @@ jest.mock("@/lib/data-hooks/useRoadmap", () => {
 jest.mock("next/router", () => {
   return { useRouter: jest.fn() };
 });
-
+jest.mock("@/lib/api-client/apiClient", () => ({
+  postTaxFilingsOnboarding: jest.fn(),
+  postTaxFilingsLookup: jest.fn(),
+}));
```

Comment:
why are we mocking these here? It doesn't appear to be used anywhere

Reply:
We need it to fix the **No User Pool in the configuration** error.